### ISSUE TITLE
License

### ISSUE THREAD

**Issue by colmartofus** [carlosbeltran/mscrdist#1]
Hi. I was wondering what the license is for the code. Is it BSD-2 Clause like the SDALF matlab code?

Also, was this C++ implementation verified against the Matlab code to give the same results? Are there any edges cases you didn't translate? 

Thanks!

**Comment by carlosbeltran** [comment]
Yes. You can assume the same license. This was indeed part of an effort to convert all SDALF into C++. My approach was to get the exact output in both Matlab and C++ (up to machine rounding errors) so one thing was to build a unit testing harness around the code . For reason I don't yet understand my colleagues didn't like this TDD approach so I discontinued the development. Thus, for sure, there is unfinished stuff.

**Comment by colmartofus** [comment]
The license is great news. 

How unfinished is the code? Unfinished as in:
1. it won't give the same results as the Matlab implementation because some steps aren't complete
or 
2. gives the same results but it needs some optimization?
